node-commons/src/main/java/org/apache/iotdb/commons/partition/ConsensusGroupId.java

```diff
@@ -17,17 +17,17 @@
  * under the License.
  */
 
-package org.apache.iotdb.consensus.common;
+package org.apache.iotdb.commons.partition;
```

Comment:
Place the ConsensusGroupId in org.apache.iotdb.commons.consensus might be better.

Reply:
Done